connectors/utils.py

```diff
@@ -352,3 +353,42 @@ def get_event_loop(uvloop=False):
             loop = asyncio.new_event_loop()
             asyncio.set_event_loop(loop)
     return loop
+
+
+class RetryStrategy(Enum):
+    CONSTANT = 0
+    LINEAR_BACKOFF = 1
+    EXPONENTIAL_BACKOFF = 2
+
+
+class UnknownRetryStrategyError(Exception):
+    pass
+
+
+def retryable(retries=1, interval=1, strategy=RetryStrategy.LINEAR_BACKOFF):
```

Comment:
neat!
I would use `retries=3` by default. No incentive to user `retryable()` if it tries only once.